hal/src/thumbv7em/clock/v2/rtc.rs

```diff
@@ -0,0 +1,99 @@
+//! RTC - Real Time Clock
+//! TODO
+//!
+//! This is a bit of a hack right now. I think it might be best if the RTC
+//! migrates into the `clock` module, since it's so integrated with OSC32KCTRL.
```

Comment:
Yeah, it doesn't really conform very well to the rest of the module. But it's fine for now.